_episodes/04-formatting.md

```diff
@@ -103,7 +103,7 @@ authors should use only:
 
 Authors should *not* use:
 
-*   sub-headings
+*   sub-titles (use H2 subheadings instead)
```

Comment:
How about being explicit about `##` in the .(R)md files here?

Reply:
Good suggestion!

Reply:
how is this @katrinleinweber https://github.com/carpentries/lesson-example/pull/225/commits/6463d780e89a7fc8127422cc6ed137a45c738dd9

Reply:
should the h2 headings be added to  the list of things to be used above?

Reply:
> should the h2 headings be added to the list of things to be used above?

Makes sense, I think.